### PR TITLE
Format updates in catalog topics folder (part 2)

### PR DESCRIPTION
**Pull request by leslietilling**
## Purpose of this pull request

This is primarily as a fix for #21 (update formats in catalog folder).

- Replaced H2/H3 infinitives with present tense verb headings (better for machine translation and findability) and sentence case (to match devdocs and adobe)
- Fixed levels for headings in some places
- Adjusted some procedures to better use numbered steps, as well as clarify navigation and simplify the instructions
- A few other misc. formats to align with MD linting rules
- Some minor editorial for consistency 

**_Note:_** To keep the PR from being too large, this is the second of a series to address #21

**_Also:_** This is not a comprehensive rewrite! Other style edits can be done at a later time.

## Affected documentation pages

https://docs.magento.com/m2/ee/user_guide/catalog/inventory-cli.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-about-order-status-reservation.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-about-product-types.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-about-sources-stocks.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-about-ssa.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-backorders.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-bulk-assign-sources.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-bulk-transfer-inventory.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-bulk-unassign-sources.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-configure-distance-priority.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-configure-inventory-management.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-configure-source-priority.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-expand-restructure-inventory.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-get-started.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-import-export.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-manage-inventory-quantities.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-merchant-type.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-migration.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-options-global.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-orders-shipment.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-product-alert-run-settings.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-product-alerts.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-product-quantity.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-product-sources.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-product-stock-options.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-shipment-create.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-sources-add.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-sources-disable.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-sources.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-stock-add.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-stock-delete.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-stock-message-scenarios.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-stock-priority.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory-stock.html
https://docs.magento.com/m2/ee/user_guide/catalog/inventory.html

## Affected Magento editions

- [x] Open Source
- [x] Commerce
- [x] B2B

## Links to Magento source code or PRs

https://github.com/magento/merchdocs/pull/305

## Additional information

Internal staging `362`

### CODE REVIEW

File: src/catalog/inventory-product-stock-options.md [hunk 1]
```diff
@@ -2,67 +2,66 @@
...
 
-1. Set the Stock Status as In Stock or Out of Stock.
+    - To modify the Notify for Quantity Below per source, clear or select the **Notify Quantity Use Default** checkbox.
 
-1. To modify the Notify for Quantity Below per source, clear or select the **Notify Quantity Use Default** checkbox. If cleared, enter the stock level that triggers notification that the item is out of stock. This amount is subtracted from the Salable Quantity at the stock level.
+      If cleared, enter the stock level that triggers notification that the item is out of stock. This amount is subtracted from the Salable Quantity at the stock level.
 
-   * Select to use Default: Magento checks the product Advanced Inventory options for configuration settings.
-   * Clear to Modify: Enter a value for the Notify Quantity, overriding Advanced Inventory and Store configuration settings.
+       Select to use Default |Magento checks the product Advanced Inventory options for configuration settings.
```
Comment (hickskenh) [inline src/catalog/inventory-product-stock-options.md]:
```suggestion
       Select to use Default - Magento checks the product Advanced Inventory options for configuration settings.
```

File: src/catalog/inventory-product-stock-options.md [hunk 2]
```diff
@@ -2,67 +2,66 @@
...
+       Clear to Modify|Enter a value for the Notify Quantity, overriding Advanced Inventory and Store configuration settings.
```
Comment (hickskenh) [inline src/catalog/inventory-product-stock-options.md]:
```suggestion
       Clear to Modify - Enter a value for the Notify Quantity, overriding Advanced Inventory and Store configuration settings.
```

File: src/catalog/inventory-product-stock-options.md [hunk 3]
```diff
@@ -71,35 +70,38 @@ Configure the quantities and additional settings per [added source]({% link cata
...
+1. Set **Backorders** to one of the following:
 
-   |Option|Description|
-   |--|--|
    | No Backorders | To not accept backorders when product is out of stock. |
    | Allow Qty Below 0 | To accept backorders when the quantity falls below zero. |
    | Allow Qty Below 0 and Notify Customer | To accept backorders when the quantity falls below zero, and notify the customer that the order can still be placed. |
 
-1. In the **Notify for Quantity Below** field, enter the stock level that triggers notification that the item is out of stock. This amount is subtracted from the Salable Quantity at the stock level.
+   For complete information on backorders, see [Configuring Backorders]({% link catalog/inventory-backorders.md %}).
```
Comment (hickskenh) [inline src/catalog/inventory-product-stock-options.md]:
```suggestion
   For more information, see [Configuring Backorders]({% link catalog/inventory-backorders.md %}).
```

File: src/catalog/inventory-product-stock-options.md [hunk 4]
```diff
@@ -71,35 +70,38 @@ Configure the quantities and additional settings per [added source]({% link cata
...
 
-1. To activate quantity increments for the product, set **Enable Qty Increments** to “Yes.” Then in the **Qty Increments** field, enter the number of the items that must be purchased to meet the requirement. For example, an item that is sold in increments of 6 can be purchased in quantities of 6, 12, 18, and so on.
+1. In the **Notify for Quantity Below** field, enter the stock level that triggers notification that the item is out of stock.
```
Comment (hickskenh) [inline src/catalog/inventory-product-stock-options.md]:
```suggestion
1. For **Notify for Quantity Below**, enter the stock level that triggers notification that the item is out of stock.
```

File: src/catalog/inventory-shipment-create.md [hunk 5]
```diff
@@ -8,40 +8,44 @@ When creating shipments, use the Source Selection Algorithm for calculated recom
...
-## To create a shipment:
+## Create a shipment
 
-1. On the Admin sidebar, choose **Sales**. Then, choose **Orders**.
+1. On the _Admin_ sidebar, go to **Sales** > **Orders**.
 
 1. Locate the order and open in **View** mode.
 
-1. If the order has been paid and invoiced, and is ready to ship, click <span class="btn">Ship</span>.
+1. If the order is paid and invoiced, and is ready to ship, click <span class="btn">Ship</span>.
```
Comment (hickskenh) [inline src/catalog/inventory-shipment-create.md]:
```suggestion
1. If the order is paid and invoiced and is ready to ship, click <span class="btn">Ship</span>.
```

File: src/catalog/inventory-shipment-create.md [hunk 6]
```diff
@@ -8,40 +8,44 @@ When creating shipments, use the Source Selection Algorithm for calculated recom
...
+      |[Distance Priority]({% link catalog/inventory-configure-distance-priority.md %})|Recommends shipments from sources closest to the shipping address based on physical distance or shortest time to deliver.|
+
+    {:.bs-callout-info}
+    **Important:** When using the Distance Priority algorithm for shipping, if routes and data does not return for the selected [Computation mode]({% link catalog/inventory-configure-distance-priority.md %}) (driving, bicycling, or walking) for a shipment, the SSA defaults to using the Source Priority. We recommend also setting the [priority for sources per stock]({% link catalog/inventory-stock-priority.md %}).
```
Comment (hickskenh) [inline src/catalog/inventory-shipment-create.md]:
```suggestion
    **Important:** When using the Distance Priority algorithm for shipping and routes and data does not return for the selected [Computation mode]({% link catalog/inventory-configure-distance-priority.md %}) (driving, bicycling, or walking) for a shipment, the SSA defaults to the Source Priority. We recommend also setting the [priority for sources per stock]({% link catalog/inventory-stock-priority.md %}).
```

File: src/catalog/inventory-shipment-create.md [hunk 7]
```diff
@@ -8,40 +8,44 @@ When creating shipments, use the Source Selection Algorithm for calculated recom
...
 
-   1. For the **Select a Source to Ship from**, select a source from the drop-down menu to send a shipment.
+    - For the **Select a Source to Ship from**, select a source from the menu to send a shipment.
```
Comment (hickskenh) [inline src/catalog/inventory-shipment-create.md]:
```suggestion
    - For  **Select a Source to Ship from**, select a source from the menu to send a shipment.
```

File: src/catalog/inventory-sources-add.md [hunk 8]
```diff
@@ -6,77 +6,79 @@ Manage inventory and order fulfillment from multiple locations with custom sourc
...
 
-   1. Enter a brief **Description** for this location for quick reference or additional details.
+    - If this inventory source is ready to use, set **Is Enabled** to `Yes`.
 
-   1. In the **Latitude** and **Longitude** fields, enter the Global Positioning System (GPS) coordinates of the facility location.
+    - Enter a brief **Description** for this location for quick reference or additional details.
 
-      To find the GPS coordinates with [Google Maps][1], enter the address in the **Search** box. Right-click the marker on the map and choose **What’s here?**.
-      The GPS coordinates appear in the details box below the street address.
+    - In the **Latitude** and **Longitude** fields, enter the Global Positioning System (GPS) coordinates of the facility location.
```
Comment (hickskenh) [inline src/catalog/inventory-sources-add.md]:
```suggestion
    - For **Latitude** and **Longitude**, enter the Global Positioning System (GPS) coordinates of the facility location.
```

File: src/catalog/inventory-sources-add.md [hunk 9]
```diff
@@ -6,77 +6,79 @@ Manage inventory and order fulfillment from multiple locations with custom sourc
...
 
       ![]({% link images/images/stores-inventory-source-general.png %}){: .zoom}
-      *General*
+      _General_
 
-1. Expand ![]({% link images/images/btn-expand.png %}){: .Inline} the **Contact Info** section, and do the following:
+1. Expand ![]({% link images/images/btn-expand.png %}) the **Contact Info** section and do the following:
 
-   1. For the **Contact Name**, enter the full name of the primary contact at the location.
+    - For the **Contact Name**, enter the full name of the primary contact at the location.
```
Comment (hickskenh) [inline src/catalog/inventory-sources-add.md]:
```suggestion
    - For **Contact Name**, enter the full name of the primary contact at the location.
```

File: src/catalog/inventory-sources-add.md [hunk 10]
```diff
@@ -6,77 +6,79 @@ Manage inventory and order fulfillment from multiple locations with custom sourc
...
 
-   1. Enter an **Email** address for contacting the location.
+    - Enter an **Email** address for contacting the location.
 
-   1. For the **Phone**, enter the area code and phone number.
+    - For the **Phone**, enter the area code and phone number.
```
Comment (hickskenh) [inline src/catalog/inventory-sources-add.md]:
```suggestion
    - For **Phone**, enter the area code and phone number.
```

File: src/catalog/inventory-sources-add.md [hunk 11]
```diff
@@ -6,77 +6,79 @@ Manage inventory and order fulfillment from multiple locations with custom sourc
...
 
-   1. In the **Fax** field, enter the area code and phone number of the fax, if available.
+    - In the **Fax** field, enter the area code and phone number of the fax, if available.
```
Comment (hickskenh) [inline src/catalog/inventory-sources-add.md]:
```suggestion
    - For **Fax**, enter the area code and phone number of the fax, if available.
```

File: src/catalog/inventory-sources-add.md [hunk 12]
```diff
@@ -6,77 +6,79 @@ Manage inventory and order fulfillment from multiple locations with custom sourc
...
+      _Contact Info_
 
-1. Expand ![]({% link images/images/btn-expand.png %}){: .Inline} the **Address Data** section, and do the following:
+1. Expand ![]({% link images/images/btn-expand.png %}) the **Address Data** section and do the following:
 
-   1. Choose the **Country**.
+    - Choose the **Country**.
 
-   1. For the **State/Province**, enter the standard abbreviation for the state or province.
+    - For the **State/Province**, enter the standard abbreviation for the state or province.
```
Comment (hickskenh) [inline src/catalog/inventory-sources-add.md]:
```suggestion
    - For **State/Province**, enter the standard abbreviation for the state or province.
```

File: src/catalog/inventory-sources-add.md [hunk 13]
```diff
@@ -6,77 +6,79 @@ Manage inventory and order fulfillment from multiple locations with custom sourc
...
 
-   1. Enter the **City**.
+    - Enter the **City**.
 
-   1. Enter the physical **Street** address.
+    - Enter the physical **Street** address.
 
-   1. For the **Postcode**, enter the ZIP or postal code.
+    - For the **Postcode**, enter the ZIP or postal code.
```
Comment (hickskenh) [inline src/catalog/inventory-sources-add.md]:
```suggestion
    - For **Postcode**, enter the ZIP or postal code.
```

File: src/catalog/inventory-sources-add.md [hunk 14]
```diff
@@ -6,77 +6,79 @@ Manage inventory and order fulfillment from multiple locations with custom sourc
...
 
-### Button Bar
+## Button bar
 
 |Button|Description|
 |--|--|
 |Back|Returns to the Manage Sources page.|
 |Reset|Restores all fields in the form to their values at the time of the last save.|
-|Save &amp; Continue|Saves all changes, and keeps the form open for further editing. Click the down arrow for these additional options:<br/>**Save &amp; Close**: Saves changes to the current record, closes the form, and returns to the Manage Sources page.<br/>**Save &amp; New**:Saves changes and closes the current record, and opens a new blank form.|
+|Save & Continue|Saves all changes, and keeps the form open for further editing. Click the down arrow for these additional options:<br/>**Save & Close** - Saves changes to the current record, closes the form, and returns to the Manage Sources page.<br/>**Save & New** - Saves changes and closes the current record, and opens a new blank form.|
```
Comment (hickskenh) [inline src/catalog/inventory-sources-add.md]:
```suggestion
|Save & Continue|Saves all changes and keeps the form open for further editing. Click the down arrow for additional options:<br/>**Save & Close** - Saves changes to the current record, closes the form, and returns to the Manage Sources page.<br/>**Save & New** - Saves changes, closes the current record, and opens a new blank form.|
```

File: src/catalog/inventory-sources-disable.md [hunk 15]
```diff
@@ -16,33 +16,31 @@ You cannot disable the Default Source. Magento uses this source for all new, imp
...
+- **Seasonal shipments** - Holidays can be an extremely busy time of the year. You may want to restrict shipping only from specific shipment locations such as warehouses to keep brick-and-mortar locations well stocked and focused on local shoppers. Or you may add new shipment locations for a limited time to handle higher rates of sales and incoming orders.
+- **Closing a location** - When closing a location for movement to new facilities or permanently, disable to stop new shipments from the location.
 
-## To disable one or more custom sources:
+## Disable one or more custom sources
 
-1. On the _Admin_ sidebar,  go to **Stores** > _Inventory_ > **Sources**.
+1. On the _Admin_ sidebar, go to **Stores** > _Inventory_ > **Sources**.
 
 1. Select the checkbox for each enabled custom source that you want to disable
```
Comment (hickskenh) [inline src/catalog/inventory-sources-disable.md]:
```suggestion
1. Select the checkbox for each enabled custom source that you want to disable.
```

File: src/catalog/inventory-about-order-status-reservation.md [hunk 16]
```diff
@@ -55,30 +55,30 @@ The following table details order and credit memo status with reservation change
...
 
-* **Shipped products**: With products purchased and shipped, Magento updates the inventory. Reservation compensations convert to on-hand inventory quantity deductions from the shipped source. The available salable quantity updates for the stock.
-* **Canceled products**: By canceling stock, Magento removes the reservation for that product. Reservation compensation is entered to the stock level to add back salable quantities for the partial cancellation of two shirts. This does not affect the inventory quantity at the source level.
-* **Credit Memo/Refunded product**: By returning stock, we need to add back to our quantities. When issuing the credit memo, we select to return to stock. Magento adds back inventory quantity to the shipped source for the product. Reservation compensations enter to clear any remaining reservations. The salable quantity recalculates against the updated quantity.
+- **Canceled products** - By canceling stock, Magento removes the reservation for that product. Reservation compensation is entered to the stock level to add back salable quantities for the partial cancellation of two shirts. This does not affect the inventory quantity at the source level.
```
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/catalog/inventory-about-order-status-reservation.md]: "stock" has a special meaning within the context of Inventory Management. I would not change that without consulting the SMEs for that module.

File: src/catalog/inventory-about-order-status-reservation.md [hunk 17]
```diff
@@ -55,30 +55,30 @@ The following table details order and credit memo status with reservation change
...
 
-![]({% link images/images/inventory/inventory-diagram-order2.png %})
+- **Credit Memo/Refunded product** - By returning stock, we need to add back to our quantities. When issuing the credit memo, we select to return to stock. Magento adds back inventory quantity to the shipped source for the product. Reservation compensations enter to clear any remaining reservations. The salable quantity recalculates against the updated quantity.
```
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/catalog/inventory-about-order-status-reservation.md]: Same as above

File: src/catalog/inventory-about-product-types.md [hunk 18]
```diff
@@ -2,12 +2,13 @@
 title: About Product Types
 ---
 
-
 Inventory Management supports inventory and order management for all product types in Magento: simple, configurable, virtual, downloadable, bundle, and grouped. Options and requirements may differ per product type for sources, stocks, and shipping.
 
-* Single Source merchants create and update product settings and quantities without requiring additional updates. All created and newly imported products automatically assign to the Default Source and Default Stock, immediately available to customers if enabled and In-Stock.
-* Multi Source merchants assign sources, quantities per source, and settings during or after product creation. Magento assigns all newly imported products to the Default Source, requiring additional edits to assign sources and quantities.
+- Single Source merchants create and update product settings and quantities without requiring additional updates. All created and newly imported products automatically assign to the Default Source and Default Stock, immediately available to customers if enabled and In-Stock.
```
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/catalog/inventory-about-product-types.md]: We would need to figure out with the PO -- in this case Community-- what the terminology should be. I think it is that way in the UI in many places, so perhaps that is why Lori did it that way.

File: src/catalog/inventory-about-sources-stocks.md [hunk 19]
```diff
@@ -25,33 +25,31 @@ Sales Channels represent entities selling your inventory, including websites, st
...
 
 ## Inventory Configurations
 
 Every product, source, and stock includes a number of options to configure for your store at the global, source, stock, and product level. For a full list of these options, see [Configuring Inventory Management]({% link catalog/inventory-configure-inventory-management.md %}).
 
 The following are important options to understand for Inventory Management:
 
-* **Out-of-Stock Threshold** sets an amount to subtract from your Salable Quantity. If you enable backorders, this value is not deducted from the Salable Quantity.
-* **Backorders** determines if products can be sold beyond a zero inventory, saving orders until restocked. When enabled, we recommend configuring the Out-of-Stock Threshold.
+- **Out-of-Stock Threshold** sets an amount to subtract from your Salable Quantity. If you enable backorders, this value is not deducted from the Salable Quantity.
```
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/catalog/inventory-about-sources-stocks.md]: I think this is OK for now.

**Comment by leslietilling** [comment]
running tests

**Pull request merged**